src/private/patient/js/patientAPI.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener('DOMContentLoaded', function () {
+});
+
+loggedAsUser = document.getElementById('loggedAsUser');
+
+window.onload = function writeYouAreLoggedAsAndUserName() {
+    let apiRequest = new XMLHttpRequest();
+    apiRequest.open("GET", "http://" + window.location.host + "/api/util/me", true);
+
+    apiRequest.responseType = 'json';
+    apiRequest.send();
```

Comment:
niewiem czy  send nie powinnien być po podpięciu handlerów. Sprawdzałaś czy to działa?

Reply:
działa